src/test/java/com/ericsson/ei/subscriptionhandler/test/SubscriptionHandlerTest.java

```diff
@@ -137,6 +138,20 @@ public void MissedNotificationTest() {
         JsonNode output = jsonResult.get("AggregatedObject");
         assertEquals(expectedOutput.toString(), output.toString());
     }
+
+
+    @Test
+    public void sendMailTest() {
```

Comment:
Can you be more specific in test function name on what you are testing, e.g. "extractEmailsTest()" or something.